ni_measurementlink_service/_internal/parameter/serialization_strategy.py

```diff
@@ -1,23 +1,34 @@
 """Serialization Strategy."""
 
-from typing import Any, Callable
+from typing import Any, Callable, Dict
 
 from google.protobuf import type_pb2
-from google.protobuf.internal import decoder
-from google.protobuf.internal import encoder
+from google.protobuf.descriptor import FieldDescriptor
+from google.protobuf.internal import decoder, encoder
+from typing_extensions import Buffer
```

Comment:
Let's not require `typing_extensions` at runtime.

```
from __future__ import annotations
...
if typing.TYPE_CHECKING:
    from typing_extensions import Buffer
```

In type aliases, you can quote types that are not available at runtime.
```
LowLevelEncoder = Callable[[Callable[["Buffer"], int], Any, None], None]
```